nexus/reconfigurator/planning/tests/output/planner_nonprovisionable_2_2a.txt

```diff
@@ -165,6 +142,27 @@ to:   blueprint 9f71f5d3-a272-4382-9154-6ea2e171a6c6
 -   nexus          67622d61-2df4-414d-aa0e-d1277265f405   expunged      fd00:1122:3344:103::22
 
 
+  sled 68d24ac5-f341-49ea-a92a-0381b52ab387 (active):
```

Comment:
Why did this sled move from "REMOVED" to "MODIFIED"?

Reply:
This test manually messes around with `blueprint_zones`: https://github.com/oxidecomputer/omicron/blob/049663724ecf779e43dad7908a16f07055a68a4e/nexus/reconfigurator/planning/src/planner.rs#L1858-L1863

Prior to this PR, "sled missing from zones and disks" was enough for it to be `REMOVED`. After this PR, it _also_ has to be missing from `sled_state`, which the test doesn't touch. I thought that was fine, but looking back at the test it explicitly wants to test diff output including a removed sled, so I changed the test to remove the decommissioned sled from `sled_state` and now it's back to REMOVED: fab82fc4c

Reply:
Makes sense. Thanks for the details.